static/monaco.js

```diff
@@ -24,4 +24,5 @@
 
 'use strict';
 //until webpack can support monaco properly it exists on the window
-module.exports = monaco;
+var monacoEditor = require('monaco-editor');
+module.exports = monacoEditor;
```

Comment:
Does anything still import this? Maybe it should just go away and everything can import `monaco-editor` directly if so.

Reply:
The custom modes still were importing this. I have moved them to use the correct require now